README.md

```diff
@@ -61,27 +69,21 @@ unless real values don't help users know what to change.
 
 ### Required access policies
 
-<!-- PERMISSIONS REQUIRED TO RUN MODULE
-If this module requires permissions, uncomment the following block and update
-the sample permissions, following the format.
-Replace the 'Sample IBM Cloud' service and roles with applicable values.
-The required information can usually be found in the services official
-IBM Cloud documentation.
-To view all available service permissions, you can go in the
-console at Manage > Access (IAM) > Access groups and click into an existing group
-(or create a new one) and in the 'Access' tab click 'Assign access'.
--->
-
-<!--
 You need the following permissions to run this module:
 
-- Service
-    - **Resource group only**
+- Account Management
+    - **Resource group**
         - `Viewer` access on the specific resource group
-    - **Sample IBM Cloud** service
+- IAM services
```

Comment:
So far, access tags support is not added in this module. 
I am not sure if that is required because these services are being created for project. Once we have separate modules for these services, then we can have this feature in those and use it here.